package.json

```diff
@@ -189,11 +192,13 @@
 		"node-wp-i18n": "1.2.3",
 		"prettier": "https://github.com/Automattic/wp-prettier/releases/download/wp-1.17.0/wp-prettier-1.17.0.tgz",
 		"puppeteer": "1.18.1",
+		"react-autosize-textarea": "7.0.0",
 		"react-test-renderer": "16.8.6",
 		"sass-loader": "7.1.0",
 		"sinon": "7.3.2",
 		"sinon-chai": "3.3.0",
 		"style-loader": "0.23.1",
+		"traverse": "0.6.6",
```

Comment:
Why `react-autosize-textarea` and `traverse` here? I don't see them used in this PR.

Reply:
Because upstream :sob: https://github.com/WordPress/gutenberg/pull/16628

Reply:
Could we use `@wordpress/editor` dependency instead? See https://github.com/Automattic/jetpack/pull/13070#discussion_r306203333

Reply:
`@wordpress/editor` OTOH requires another package (`@wordpress/server-side-render`, and counter-intuitively, this isn't really about SSR, but about rendering a preview of PHP-rendered blocks in the editor. IIRC, this one had a number of implicit DOM dependencies :roll_eyes: ).

Reply:
Ah, wasn't so bad after all: 62221b9 and cbe7dd8.

Reply:
I'll see if I can remove `react-autosize-textarea` and `traverse` now.

Reply:
af61737 and bbaf7ae :heavy_check_mark: